internal/git/git.go

```diff
@@ -166,8 +169,28 @@ func (r *Repo) CheckoutBranch(opts *CheckoutBranch) (string, error) {
 		args = append(args, "-b")
 	}
 	args = append(args, opts.Name)
-	if _, err := r.Git(args...); err != nil {
-		return "", err
+	res, err := r.Run(&RunOpts{
+		Args: args,
+	})
+	if err != nil {
+	    return "", err
+	}
+	if res.ExitCode != 0 {
+	    _, _ = fmt.Fprint(os.Stderr,
```

Comment:
updated to return an error string, but it's a bit repetitive because it's nested -
`error: failed to checkout branch "test test": failed to checkout branch "test test": fatal: 'test test' is not a valid branch name.`

should i change one of the messages? not sure if there's a correct pattern here for go

Reply:
Yeah I think it's fine to delete the the wrapping in `stack_branch.go`. In generally, being a bit more consistent about error wrapping is something I want to get better about.